IfcRail & IfcRailway Import BIMQ 2021-07-21T06:54:00.4321802Z (#163)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/c/CaseOfApplication_2uDCY_lmr2KBMHe8MLnaZT/Documentation.md

> Where is the check rail used? In turnout, track dilatation, etc.

Reviewer: ```suggestion
Usage of the check rail in turnout, track dilatation, etc.
```

Reviewer 2: This is an orphan. Removed: https://github.com/bSI-InfraRoom/IFC-Specification/pull/163/commits/0f6a2672e05cf6089ccc9c1aa1e68ff3a7b4a16e
